.changeset/lazy-nails-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
+---
+
+Added `fixedFirstColumns` prop to `DataTable` so that multiple columns can be fixed
```

Comment:
```suggestion
- Added a `fixedFirstColumns` prop to `DataTable` so that multiple columns can be fixed
- Deprecated the `DataTable` `fixedFirstColumn` prop 
```

Reply:
Looks like this caused a bit of a kerfuffle in the [generated changelog](https://github.com/Shopify/polaris/pull/6999). Can we reduce this to a single line without bullets? Not as fancy, but might improve how changesets renders it

<img width="811" alt="Screen Shot 2022-08-26 at 6 29 08 AM" src="https://user-images.githubusercontent.com/11774595/186905076-464ec2fb-7918-4c2f-b87a-fbe491ec124d.png">

Reply:
You can just modify the changeset file directly (`.changeset/lazy-nails-relax.md`) no command needed 👍